content/docs/command-reference/run.md

```diff
@@ -1,13 +1,13 @@
 # run
 
-Create a pipeline _stage_ in
-[`dvc.yaml`](/doc/user-guide/dvc-files-and-directories) from a given command,
-and execute the command.
+Helper command to create or update _stages_ in
+[`dvc.yaml`](/doc/user-guide/dvc-files-and-directories). Requires a name and a
+command.
 
 ## Synopsis
 
 ```usage
-usage: dvc run [-h] [-q | -v] [-d <path>] [-n <name>] [-o <path>]
+usage: dvc run [-h] [-q | -v] -n <name> [-d <path>] [-o <path>]
```

Comment:
If -n is required, why is is printed in `[]` here?

Reply:
@jorgeorpinel Because it is not strictly required, we have a hiddlen `--single-stage` hack for which it is not required.

Reply:
Ok good to know.

P/s should the `-h` output

```
  -n NAME, --name NAME  Stage name.
```

Say it's (usually) required?